definitions/npm/just-safe-get_v1.3.x/flow_v0.98.x-/just-safe-get_v1.3.x.js

```diff
@@ -0,0 +1,11 @@
+declare module 'just-safe-get' {
+  declare export default function get(
+    obj: mixed,
+    key: string | Array<string>
+  ): mixed;
+
+  declare export default function get<TReturn>(
+    obj: mixed,
+    key: string | Array<string>
+  ): TReturn;
+}
```

Comment:
This comes up often in other libraries, whether to offer `mixed` or `any` or your strategy of a parameterized generic.

I fall on the "keep it useful and call it `any`" so you're not doing a bunch of type refinements all over the place. Something as low-level as `get` may get very frustrating to use if you have to provide the type always.

Just something to consider. I'll merge it in as-is because I have no vested interest in the usage of this particular libdef. 😉

Reply:
Yeah, that's a reasonable point.  It isn't always clear where to fall on the trade-off between soundness and ease-of-use.

I'm partial to the parameterized generic as it defers the concrete typing to the consumer without necessitating any other type refinements.  But if that isn't a standard way of handling this, it may throw some folks off.

Reply:
Yeah it's a long-standing disagreement within the flow community. :-)

I don't think there's really a standard, though I would argue I see more cases of type looseness than correctness. That's at least the case for libraries like `axios` and `ramda`.

Shrug. There's no right answer. You do what's good for you, and if people dislike they can argue. 😆